components/server/src/auth/bearer-authenticator.ts

```diff
@@ -51,10 +51,10 @@ export class BearerAuth {
                     res.status(401).send(e.message);
                     return;
                 }
-                throw e;
+                return next(e);
```

Comment:
@AlexTugarev Does this may sense? Before, those errors would end up as "UnhandledPromiseRejections".

Reply:
In general, yes. As I read the the generic error handler, this would result in a code 500 response ✔️

Reply:
Thx for confirming! :pray: